editor/store-persist.js

```diff
@@ -41,7 +50,7 @@ export default function storePersist( reducerKey, storageKey = DEFAULT_STORAGE_K
 
 			store.dispatch( {
 				type: 'REDUX_REHYDRATE',
-				payload: persistedState,
+				payload: beforeRehydrate( persistedState ),
```

Comment:
It looks a bit like a middleware inside another middleware :) I was wondering if that would be possible to create `mobileMiddleware` instead and intercept `REDUX_REHYDRATE` action and apply modification there. Something like this:

```js
const mobile = store => next => action => {
  if ( action.type ===  'REDUX_REHYDRATE' ) {
    return next( {
      type: 'REDUX_REHYDRATE',
      payload: disableIsSidebarOpenedOnMobile( action.payload ),
    } );
  }
  return next( action )
}
```
It probably would have to be added in the chain of middlewares before this one. This way we would have the mobile logic hidden behind its own middleware. We could avoid using:
```js
beforeRehydrate: disableIsSidebarOpenedOnMobile,
```
which is an elegant solution for this complex case, but at the same time difficult to follow.